static/styles/portico.css

```diff
@@ -1259,6 +1259,10 @@ input.new-organization-button {
     position: relative;
 }
 
+.app-main {
+    padding: 0px;
+}
```

Comment:
@utkarsh23 why did you make this change?  It affects a lot of pages, and I'd worry it'll make other pages look wrong.

Reply:
![ss](https://user-images.githubusercontent.com/12075549/36749510-ff0cdc1e-1c20-11e8-840f-973dba042c4c.png)

On left and right edges there's some padding right, I made this change to make the padding 0px.
Since this change might affect other pages also, I'll remove this piece of code and commit again.